ligo/src/types.mligo

```diff
@@ -87,6 +87,8 @@ type migration_status =
   | MigratingTo of address
   | MigratedTo of address
 
+type vote_type = bool
```

Comment:
Can you add a comment to clarify which value means "voted for" and which "voted against"?

Just to have it cleared up somewhere for whoever reads this next